cli/actions/config.go

```diff
@@ -41,15 +38,13 @@ func (config configActions) Apply(ctx context.Context, args ApplyArgs) error {
 		return fmt.Errorf(`file must be of type "Config"`)
 	}
 
-	var configurationResource openapi.ConfigurationResource
-	deepcopy.DeepCopy(fileContent.Definition(), &configurationResource)
-	deepcopy.DeepCopy(fileContent.Definition().Spec, &configurationResource.Spec)
-
-	request := config.client.ResourceApiApi.UpdateConfiguration(ctx, "current")
-	request = request.ConfigurationResource(configurationResource)
-
-	_, res, err := config.client.ResourceApiApi.UpdateConfigurationExecute(request)
+	url := fmt.Sprintf("%s/%s", config.resourceClient.BaseUrl, "current")
```

Comment:
To avoid hardcoding the `current` id in other places of the code, we can create a constant for it, it makes sense?

```go 
// near the file beginning
const configID = "current"

// here
url := fmt.Sprintf("%s/%s", config.resourceClient.BaseUrl, configID)
```